JavaFiddle-ejb/src/main/java/Test.java

```diff
@@ -0,0 +1,67 @@
+
+
+import ru.javafiddle.core.ejb.ProjectBean;
+import ru.javafiddle.jpa.entity.*;
+import ru.javafiddle.core.ejb.UserBean;
+import ru.javafiddle.core.ejb.GroupBean;
+
+import javax.faces.event.SystemEvent;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+import java.io.UnsupportedEncodingException;
+import java.security.NoSuchAlgorithmException;
+
+
+/**
+ * Created by artyom on 06.02.16.
+ */
+public class Test {
```

Comment:
It was for testing , we don't need it. 
23 марта 2016 г., в 1:48, alexander1leonov <notifications@github.com> написал(а):

> In JavaFiddle-ejb/src/main/java/Test.java:
> 
> > +import ru.javafiddle.jpa.entity.*;
> > +import ru.javafiddle.core.ejb.UserBean;
> > +import ru.javafiddle.core.ejb.GroupBean;
> > +
> > +import javax.faces.event.SystemEvent;
> > +import javax.persistence.EntityManager;
> > +import javax.persistence.EntityManagerFactory;
> > +import javax.persistence.Persistence;
> > +import java.io.UnsupportedEncodingException;
> > +import java.security.NoSuchAlgorithmException;
> > +
> > +
> > +/**
> > + * Created by artyom on 06.02.16.
> > + */
> > +public class Test {
> Do we need this class at all? I hardly think so.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>